Add  missing movies and Animes

Arial-Z opened the pull request:
Movies added:
- Demon Slayer -Kimetsu no Yaiba- The Movie: Mugen Train
- Kabaneri of the Iron Fortress: The Battle of Unato
- Made in Abyss: Journey's Dawn
- Made in Abyss: Wandering Twilight
- Made in Abyss: Dawn of the Deep Soul
- My Hero Academia : Two Heroes
- My Hero Academia : Heroes Rising
- The Garden of Sinners (as separate movies)
- The Irregular at Magic High School: The Girl Who Summons the Stars
- Sword Art Online: The Movie – Ordinal Scale

Animes added :
- Ace of Diamond
- Blue Exorcist
- Kamisama Kiss
- Knights of Sidonia
- Major
- Mobile Suit Gundam 00
- Natsume's Book of Friends
- Showa Genroku Rakugo Shinju
- The Demon Girl Next Door
- The Outcast
- The Yakuza's Guide to Babysitting
- Wagnaria!!

Animes synonyms added to match MAL title :
- Darker Than Black > Darker than Black: Kuro no Keiyakusha
- Kaguya-sama: Love is War > Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen
- My Isekai Life: I Gained a Second Character Class and Became the Strongest Sage in the World! > Tensei Kenja no Isekai Life: Dai-2 no Shokugyou wo Ete, Sekai Saikyou ni Narimashita

reconman commented:
You were still waiting for review, right? Was pretty busy the last few weeks so I forgot.

Arial-Z commented:
yeah, but that okay take your time :)

Arial-Z commented:
Added
- "Gintama: The Movie"
- "Gintama: The Movie: The Final Chapter: Be Forever Yorozuya"
- Added MAL synonym for "KonoSuba: God's Blessing on this Wonderful World! Legend of Crimson"

reconman commented on movies-tmdb.en.yaml:
```diff
@@ -18,17 +18,178 @@ entries:
+      - season: 1
+        anilist-id: 97918
+
+  - title: "Love, Chunibyo & Other Delusions ! -Take On Me"
```
```suggestion
  - title: "Love, Chunibyo & Other Delusions! -Take On Me"
```

reconman commented on movies-tmdb.en.yaml:
```diff
@@ -18,17 +18,178 @@ entries:
+      - season: 1
+        anilist-id: 100643
+
+  - title: "My Hero Academia : Two Heroes"
```
```suggestion
  - title: "My Hero Academia: Two Heroes"
```

reconman commented on movies-tmdb.en.yaml:
```diff
@@ -18,17 +18,178 @@ entries:
+    seasons:
+      - season: 1
+        anilist-id: 100723
+  - title: "My Hero Academia : Heroes Rising"
```
```suggestion
  - title: "My Hero Academia: Heroes Rising"
```

reconman reviewed (changes requested):
Some incorrect titles with extra spaces.

reconman commented:
Most of your mappings problably work out-of-the box using PlexAniSync's search function, but I'll add them anyways just in case.

Arial-Z commented:
Done and thank you
